server/src/components/calendar.tsx

```diff
@@ -31,73 +30,83 @@ interface CalendarEvent {
 
 const events: CalendarEvent[] = [
   {
-    id: 0,
+    id: '1',
     title: 'Board meeting',
-    start: new Date(2024, 7, 29, 9, 0, 0), // August 29, 2024, 9:00 AM
-    end: new Date(2024, 7, 29, 13, 0, 0), // August 29, 2024, 1:00 PM
+    start: new Date(2024, 7, 29, 9, 0, 0),
+    end: new Date(2024, 7, 29, 13, 0, 0),  
   },
   {
-    id: 1,
+    id: '2',
     title: 'MS training',
     allDay: true,
-    start: new Date(2024, 7, 29, 14, 0, 0), // August 29, 2024, 2:00 PM
-    end: new Date(2024, 7, 29, 16, 30, 0), // August 29, 2024, 4:30 PM
-  },
-  {
-    id: 2,
-    title: 'Team lead meeting',
-    start: new Date(2024, 7, 29, 8, 30, 0), // August 29, 2024, 8:30 AM
-    end: new Date(2024, 7, 29, 12, 30, 0), // August 29, 2024, 12:30 PM
+    start: new Date(2024, 7, 29, 14, 0, 0),
+    end: new Date(2024, 7, 29, 16, 30, 0),
   },
-  {
-    id: 11,
-    title: 'Birthday Party',
-    start: new Date(2024, 7, 30, 7, 0, 0), // August 30, 2024, 7:00 AM
-    end: new Date(2024, 7, 30, 10, 30, 0), // August 30, 2024, 10:30 AM
-  }
 ];
 
 const styles = {
```

Comment:
Let's use tailwind so that it's consistent with other styling work. If you feel that it clutters your markup, encapsulate it inside a new component.

Reply:
I added tailwind css as per your suggestions.